src/helpers/cleandata/removeResourceId.js

```diff
@@ -0,0 +1,26 @@
+// This function removes the id from an object
```

Comment:
Let's rename the folder as `cleanData` to be consistent with our filenaming conventions